### DISCUSSION TITLE
copilot is not working-我的copilot无法使用了

### DISCUSSION THREAD

**Discussion by CG-man** [community/community#112414] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

我的copilot无法使用， 我明明在2023年8月份是有订阅年费的copilot服务的，但是现在我的vscode和neovim的插件都无法使用，
我的vscode的插件提示：
```
[auth] Invalid copilot token: missing token: 403
```
我的neovim的copilot插件提示：
```
Copilot: Not authorized
```
这些问题导致我无法正常使用这个工具

-------------------------------------

My copilot cannot be used. I had copilot service with an annual subscription fee in August 2023, but now my vscode and neovim plugins cannot be used. 
The plugins of my vscode prompt: 
```
[auth] Invalid copilot token: missing token: 403
```

My neovim copilot plugin Tips: 
```
Copilot: Not authorized
```

These problems prevented me from using the tool properly
<img width="986" alt="image" src="https://github.com/community/community/assets/12809498/a064ede3-4453-4ff8-a25a-95ae4060572c">

**Comment by xingbofeng** [comment]
解决了吗楼主？遇到同样的问题，刚花了100刀开了一年就这样

**Reply by CG-man** [reply]
解决了，不知道为何我的copilot被登陆过cocopilot的地址，具体我不清楚，不过我清除了相关的记录解除了关联登陆，并且给github管理员截图后给我解决了哈

**Reply by xingbofeng** [reply]
是联系了管理员，建立了工单解决的么

**Reply by bestZwei** [reply]
怎么联系管理员，一样的问题
